src/breaker/ConsecutiveBreaker.ts

```diff
@@ -0,0 +1,27 @@
+import { CircuitState } from '../CircuitBreakerPolicy';
+import { IBreaker } from './Breaker';
+
+/**
+ * ConsecutiveBreaker breaks if more than `threshold` exceptions are received
+ * over a time period.
+ */
+export class ConsecutiveBreaker implements IBreaker {
+  private count = 0;
+
+  constructor(private readonly threshold: number) {}
+
+  /**
+   * @inheritdoc
+   */
+  public success() {
+    this.count = 0;
+    return CircuitState.Open;
```

Comment:
According to the `IBreaker` contract this shouldn't return anything and should take an argument.
TS should warn you about this though...?